spyder/plugins/projects/widgets/qcookiecutter.py

```diff
@@ -0,0 +1,419 @@
+# -*- coding: utf-8 -*-

```

Comment:
I don't like the name of this file too much. What about simply naming it `cookiecutter_qt.py`?

Reply:
Also, please change the line endings of this file to LF.

Reply:
I really prefer `qcookiecutter` to `cookiecutter_qt`

Reply:
Not going to change.

Reply:
Ok, let's leave it then. However, you haven't changed the line endings of this file to LF, as I mentioned above.